tpmlib/crb.c

```diff
@@ -36,7 +36,7 @@
 
 #define REGISTER(l, r)		(((l) << 12) | (r))
 
-static u8 locality = TPM_NO_LOCALITY;
```

Comment:
`TPM_NO_LOCALITY` is zero, the compiler puts it into BSS anyway.  So the changes here and in `tis.c` aren't necessary, but might as well stay.

Reply:
No, it isn't: https://github.com/TrenchBoot/secure-kernel-loader/blob/master/tpmlib/tpm.h#L25